src/engine/Engine.cpp

```diff
@@ -2105,11 +2105,3 @@ void Engine::setConstraintViolationThreshold( unsigned threshold )
 {
     _smtCore.setConstraintViolationThreshold( threshold );
 }
-
```

Comment:
Why are these lines removed?

Reply:
Leftovers from when we were using Makefiles (and not CMake)